### DISCUSSION TITLE
my ip got banned, maybe the automatic monitor feature is making too many requests

### DISCUSSION THREAD

**Discussion by l1uk** [abraunegg/onedrive#1376] [Q&A] [answered]
after using the client for two days i can't connect to azure servers with my ip (i get a timeout), the problem goes away if i use a vpn or connect from another network. Any ideas on how to get my ip off the blacklist?

**Comment by abraunegg** [comment, marked as answer]
@l1uk 
> after using the client for two days i can't connect to azure servers with my ip (i get a timeout), the problem goes away if i use a vpn or connect from another network. Any ideas on how to get my ip off the blacklist?

So some housekeeping - this is not an issue, its a question - please follow the support guidelines.

Secondly, what client version are you using? What error message (if any) are you getting?

I find it *very* odd that Azure is banning you .. information / evidence please.

**Reply by l1uk** [reply]
Thank you for the fast response. I will post the output of the same command executed on two different networks (i can browse onedrive via browser on both)
Version:`onedrive v2.4.10-0~202102221319~ubuntu20.04.1`

1. Output of `onedrive --synchronize` on network 1, where everything works just fine:
 ```
 Configuring Global Azure AD Endpoints
 Initializing the Synchronization Engine ...
 Syncing changes from OneDrive ...
 Downloading file etc...
```

2. Output of `onedrive --synchronize` on network 2:
```
Cannot connect to Microsoft OneDrive Service - Network Connection Issue

ERROR: Microsoft OneDrive API returned an error with the following message:
  Error Message:    Timeout was reached on handle 55D5731C7E90
  Calling Function: testNetwork()

Unable to reach Microsoft OneDrive API service, unable to initialize application
```
i'm thinking my ip got into a blacklist or something... when i installed the client 2 days ago everything worked fine on both networks

**Reply by abraunegg** [reply]
@l1uk 
> `Cannot connect to Microsoft OneDrive Service - Network Connection Issue`

So - this will be a local environment issue - a problem on *your* Linux system. Microsoft / Azure is most likely not blocking you. Most likely cause:
* A firewall rule blocking HTTPS/DNS requests
* Slow DNS response

Refer to: https://github.com/abraunegg/onedrive/wiki/Frequently-Asked-Questions#cannot-connect-to-microsoft-onedrive-service---network-connection-issue for further information

**Reply by l1uk** [reply]
To make a successful request i have to set a timeout of 25s. Definitely a dns problem. Thank you!

**Reply by abraunegg** [reply]
@l1uk 
You need to fix your DNS ... the client requires a 5 second DNS response time.

**Reply by abraunegg** [reply]
@asdf8dfafjk 
Please read: https://github.com/abraunegg/onedrive/wiki/Frequently-Asked-Questions#cannot-connect-to-microsoft-onedrive-service---network-connection-issue
